CoseSignTool.Tests/CoseSignTool.Tests.csproj

```diff
@@ -15,6 +15,7 @@
 		<!-- other property settings -->
 		<ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
 		<TestingPlatformShowTestsFailure>true</TestingPlatformShowTestsFailure>
+		<PlatformTarget>x64</PlatformTarget>

```

Comment:
This may run into trouble building on the MacOS agent. We'll have to check the workflow results to make sure.
Why do we need this here when we didn't before?

Reply:
It was added to enable the tests to run on my machine. Removed it.